src/reader/reader.js

```diff
@@ -17,6 +17,18 @@ class Reader extends Component {
     this.state = {ready: false}
   }
 
+  componentDidMount() {
+    const ua = window.navigator.userAgent
+    if (ua.indexOf("iPhone") > -1 || ua.indexOf("iPad") > -1) {
+      window.addEventListener('touchstart', () => {
+        const audioTags = Array.from(document.getElementsByClassName("react-audio-player "))
+        audioTags.forEach((a) => {
+          if (!a.ended && !a.playing) a.play()
+        })
+      })
+    }
+  }
+
```

Comment:
Seems reasonable, but I think you might see interesting issues as you start to reuse audio tags and change the source across different platforms just like how there are issues in iOS right now. My two cents is that it would require a lot of effort and testing to get right.